docs/OpenWeatherMapAPI.md

```diff
@@ -4,7 +4,7 @@
 
 ### API Endpoint
 
-- [Get 5 Day Weather Forecast with 3-hour Step](https://api.openweathermap.org/data/2.5/forecast?lat={lat}&lon={lon}&appid={API key})
+- [Get 5 Day Weather Forecast with 3-hour Step](http://api.openweathermap.org/data/2.5/forecast)
```

Comment:
Sorry, I missed it. I think that we shouldn't make it as markdown link. Just leave URL and description as it's
In real-world scenario your reader won't hit the link, they will copy URL in order to save it to `.env`. I wanna make it convenient